snc.sh

```diff
@@ -128,7 +135,7 @@ replace_pull_secret ${INSTALL_DIR}/install-config.yaml
 ${YQ} eval ".sshKey = \"$(cat id_ecdsa_crc.pub)\"" --inplace ${INSTALL_DIR}/install-config.yaml
 
 # Create the manifests using the INSTALL_DIR
-${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} create manifests
+OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE ${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} create manifests
```

Comment:
Is this doing the right thing when `CLUSTER_PATCH` isn't set? May be better to export `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` after we set it?

Reply:
yes it is still doing the correct thing even cluster is not set, to get the manifest for override image.